OpenSim/Moco/MocoGoal/MocoOrientationTrackingGoal.cpp

```diff
@@ -82,9 +82,21 @@ void MocoOrientationTrackingGoal::initializeOnModelImpl(const Model& model)
         // Check that the reference state names match the model state names.
         checkLabelsMatchModelStates(model, statesTableToUse.getColumnLabels());
 
+        // Check that all coordinates are in the table
+        const CoordinateSet& coords = model.getCoordinateSet();
+        std::vector<std::string> columns = statesTableToUse.getColumnLabels();
```

Comment:
Minor: you could use `auto` here: `auto columns = statesTableToUse.getColumnLabels();`